proto/3rdparty/crates/BUILD.grpc-compiler-0.6.2.bazel

```diff
@@ -72,7 +72,7 @@ rust_binary(
     version = "0.6.2",
     deps = [
         ":grpc_compiler",
-        "@rules_rust_proto__protobuf-2.8.2//:protobuf",
-        "@rules_rust_proto__protobuf-codegen-2.8.2//:protobuf_codegen",
+            "@rules_rust_proto__protobuf-2.18.2//:protobuf",
+            "@rules_rust_proto__protobuf-codegen-2.18.2//:protobuf_codegen",
     ],
```

Comment:
This is not the indentation that would be created by the `:crates_vendor` BUILD file generator.

I see _"missing merge resolve"_ (7e363513bf98cad7a149046afc816e08564a1b02) in the commit history of this PR so I infer this result is from a badly resolved merge conflict that you edited manually, not what `:crates_vendor` actually generated.

It's almost never correct to try manually resolving merges inside generated code. You just regenerate the generated file.

Also from what's visible in that merge, you originally created this update against a very old commit of this repo (at least 4 weeks old), before later rebasing it at least once and then doing at least 2 merges between the upstream branch and your branch. That is not going to work; there have been other crates updated by other contributors in the meantime. You can tell because the Cargo.Bazel.lock in this PR is invalid and makes `bazel run //proto/3rdparty:crates_vendor` fail,  whereas it's supposed to work and it works currently on `main`.

```console
error: failed to select a version for `protobuf`.
    ... required by package `grpc v0.6.2`
    ... which satisfies dependency `grpc = "^0.6.2"` (locked to 0.6.2) of package `direct-cargo-bazel-deps v0.0.1 (/tmp/.tmpgVYSMl)`
versions that meet the requirements `>=2.8.0, <2.10.0` are: 2.8.2, 2.8.1, 2.8.0

all possible versions conflict with previously selected packages.

  previously selected package `protobuf v2.18.2`
    ... which satisfies dependency `protobuf = "^2.18.2"` (locked to 2.18.2) of package `direct-cargo-bazel-deps v0.0.1 (/tmp/.tmpgVYSMl)`

failed to select a version for `protobuf` which could resolve this conflict
```

Reply:
I will make the changes from scratch as suggested. I wasn't aware that these files were auto-generated - will use the mentioned bazel target for the update.